core/templates/domain/recommendations/exploration-recommendations-backend-api.service.ts

```diff
@@ -49,7 +49,7 @@ export class ExplorationRecommendationsBackendApiService {
   constructor(
     private http: HttpClient) { }
 
-  getRecommendedSummaryDicts(
+  async getRecommendedSummaryDictsAsync(
```

Comment:
I see more occurrences of this method. Have you made changes in all the files where this is used?
![image](https://user-images.githubusercontent.com/11008603/103452000-da1b0d00-4cf0-11eb-95d1-45c1ef9e9ed9.png)

Reply:
Actually this function is of another service so basically two functions of different services have same name so I didn't changed these. @kevintab

Reply:
ah I see. Thanks @atpug22 for clarifying!